### PR TITLE
fix: answer of HTML quiz Q21 is incorrect

### PR DESCRIPTION
**Pull request by oorkan**
## PR Checklist

This PR is ready for review and meets the requirements set out
in [Suggestion how to contribute](CONTRIBUTING.md)

### DOD

- [ ] I have added new quiz{'s}
- [ ] I have added new reference link{'s}
- [x] I have made small correction/improvements

### Changes / Instructions

The answer to HTML quiz question 21 (i.e., _"unordered list inside a nav element"_) is incorrect. According to W3C:
[**Source**](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-nav-element.html)

Not all groups of links on a page need to be in a `nav` element — the element is primarily intended for sections that consist of major navigation blocks. In particular, it is common for footers to have a short list of links to various pages of a site, such as the terms of service, the home page, and a copyright page. The `footer` element alone is sufficient for such cases; while a `nav` element can be used in such cases, it is usually unnecessary.

According to MDN:
[**Source**](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes)

It's not necessary for all links to be contained in a <nav> element. <nav> is intended only for a major block of navigation links; typically the [<footer>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) element often has a list of links that don't need to be in a [<nav>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element.

Further reading:
**IMPORTANT!** This last source is not "official" so should be considered with care. However, I thought it worth mentioning because it directly touches the topic. 

Source: https://www.a11ymatters.com/article/intro-html5-sectioning-elements/#the-nav-element

Below is an example on using multiple `<nav>`s for primary and secondary navigation:

```html
<header>
  <h1>a11ymatters</h1>
  <!-- primary navigation -->
  <nav role="navigation">
    <ul>
      <li><a href="/articles">Articles</a></li>
      <li><a href="/projects">Projects</a></li>
      <li><a href="/contact">Contact</a></li>
    </ul>
  </nav>
</header>
<main>
  <!-- secondary navigation -->
  <nav role="navigation">
    <ul>
      <li><a href="projects/web/">Web</a></li>
      <li><a href="projects/graphic/">Graphic</a></li>
      <li><a href="projects/dev/">Development</a></li>
    </ul>
  </nav>
</main>
<footer>
  <!-- footer links -->
</footer>
```

Takeaways
...
- It’s unnecessary to use `<nav>` in `<footer>`, add the links in a `<ul>` and that’s enough.
...

Summing up, I believe the correct is just _"unordered list"_.

Thank you!

### CODE REVIEW

**Review by Ebazhanov — APPROVED** [review]
Thank you 🔥
